introduction.tex

```diff
@@ -98,8 +98,24 @@ \section{Terminology}
 
 \section{Context}
 
-This document is written to work with all RISC-V ISA extensions ratified
-through 2021.
+\begin{steps}{This specification attempts to support all RISC-V ISA extensions
+that have, roughly, been ratified through the first half of 2023.  In
+particular, though, this specification specifically addresses features in the
+following extensions:}
+\item A % Mention atomics in debug mode section
+\item C % Mentioned in single-word progbuf context
+\item D % Access register abstract register numbers.
+\item F % Access register abstract register numbers.
+\item H % Various trigger bits
+\item Sm1p13 % E.g. exceptions
+\item Ss1p13 % Mention S-Mode in various trigger stuff
```

Comment:
I would say 1.12.  Although we have early hints of the content, I haven't even seen a draft of 1.13 yet.

Reply:
There's nothing expected in 1.13 that would have any Debug impact.

Reply:
So saying 1.12 (which is my preference also) is OK here? I only wrote 1.13 because that was in Greg's e-mail.

Reply:
On the one hand one _could_ only say support up through 1.12, but that will then leave people wondering whether Debug 1.0 supports up through 1.13 (and 1.13 will be frozen well before Debug gets ratified).

Reply:
I guess that I'm OK with either.  If I see something shocking in 1.13 then I may raise a red flag later.  But it sounds like that won't be necessary.  So let's just stick with 1.13.

Reply:
Great. We'll stick with 1.13.